tools/vcli/bw/repl/repl.go

```diff
@@ -88,10 +89,50 @@ func SimpleReadLine(done chan bool) <-chan string {
 	return c
 }
 
+// startProfilingIfEnabled starts profiling for the current call to runBQLFromFile if profilingEnabled is true,
+// returning the two files to which the profiling metrics will be printed.
+func startProfilingIfEnabled(profilingEnabled bool) (*os.File, *os.File) {
+	if !profilingEnabled {
+		return nil, nil
+	}
+
+	cpuProfile, err := os.Create("cpuprofile")
+	if err != nil {
+		fmt.Println(err)
+		fmt.Println("Profiling for current BQL file was aborted.")
+		return nil, nil
+	}
+	memProfile, err := os.Create("memprofile")
+	if err != nil {
+		fmt.Println(err)
+		fmt.Println("Profiling for current BQL file was aborted.")
+		return nil, nil
+	}
+	pprof.StartCPUProfile(cpuProfile)
+
+	return cpuProfile, memProfile
+}
+
+// stopProfilingIfEnabled stops profiling for the current call to runBQLFromFile if profilingEnabled is true,
+// writing the final profiling metrics to the correspondent files and closing them.
+func stopProfilingIfEnabled(profilingEnabled bool, cpuProfile, memProfile *os.File) {
```

Comment:
Usually, it is not a good practice to receive boolean parameters because it is usually passed as a literal which is not very meaningful.
Would you mind having  a parameter struct here?
then when calling it would read stopProfilingIfEnabled(params{profilingEnabled: True...})

Reply:
I see your point, it is indeed more meaningful to pass something like `profilingEnabled: True` using a parameter struct. By the end, this function was changed and does not receive a boolean anymore, so the code above is outdated.